OpenRA.Game/Graphics/ChromeProvider.cs

```diff
@@ -109,15 +111,38 @@ public static Sprite GetImage(string collectionName, string imageName)
 				return null;
 
 			// Cached sheet
-			Sheet sheet;
+			Sheet sheet = null;
 			if (cachedSheets.ContainsKey(mi.Src))
 				sheet = cachedSheets[mi.Src];
 			else
 			{
 				using (var stream = fileSystem.Open(mi.Src))
-					sheet = new Sheet(SheetType.BGRA, stream);
-
-				cachedSheets.Add(mi.Src, sheet);
+				{
+					if (Png.Verify(stream))
+						sheet = new Sheet(SheetType.BGRA, stream);
+					else if (palette != null)
+					{
+						foreach (ISpriteLoader loader in Game.ModData.SpriteLoaders)
+						{
+							ISpriteFrame[] frames;
+							TypeDictionary metadata;
+							if (loader.TryParseSprite(stream, out frames, out metadata))
+							{
+								if (frames.Length > 0)
+								{
+									// will use only first frame if multiple frames available
```

Comment:
Comments should start with an upper case letter: `Will only use the first frame if multiple frames are available`